Controller/UserController.php

```diff
@@ -63,6 +69,8 @@ public function __construct(
         string $form,
         Handler $handler,
         UserRepository $userRepository,
+        BreadCrumbBuilder $breadcrumbBuilder,
```

Comment:
if you implement it like this you should change the milestone to major version since this is currently not BC

Reply:
true, changed